src/sentry/tasks/integrations/link_all_repos.py

```diff
@@ -71,9 +70,7 @@ def link_all_repos(
     for repo in repositories:
         try:
             config = get_repo_config(repo, integration_id)
-            integration_repo_provider.create_repository(
-                repo_config=config, organization=organization
-            )
+            integration_repo_provider.create_repository(repo_config=config, organization=rpc_org)
```

Comment:
This task currently doesn't have much in the way of error handling. Should an RPC call fail the `link_all_repos` task will not be retried. Should we add retry mechanics for RPC failures?